File: praktikum_6/PRAK603-2410817210004-Rahmad.py
baris=int(input())
kolom=int(input())
if baris!=kolom:
    print('jumlah tidak sama')
else :
    matriks=[]

    for i in range(2):
        baris_matriks = []
        for j in range(kolom):
            elemen = int(input(" "))
            baris_matriks.append(elemen)
        matriks.append(baris_matriks)
    hasil=[]
    for i in range(1):
        for j in range(kolom):
            hasil.append(matriks[i][j]*matriks[i+1][j])

for row in hasil :
    print(row ,end=" ")

File: praktikum_6/PRAK603-2410817210004-Rahmad.c
#include <stdio.h>
int main(){
    int n1,n2;
    scanf("%d",&n1);
    scanf("%d",&n2);

    if(n1!=n2){
        printf("Jumlah tidak sama");
    }else{
    int matriks[1][n2=n1];
        for(int i=0;i<1;i++){
        for(int j=0;j<n1;j++){
            scanf("%d",&matriks[i][j]);
        }
        }
        for(int i=1;i<2;i++){
        for(int j=0;j<n2;j++){
            scanf("%d",&matriks[i][j]);
        }
        }
        int hasil[n1=n2];
        for(int j=0;j<n2;j++){
        for(int i=0;i<1;i++){
        hasil[j]=matriks[i][j]*matriks[i+1][j];
        }
        }
        for(int i=0;i<n1;i++){
            printf("%d ",hasil[i]);
        }
        
        
        
    }

}